config/initializers/gitlab.rb

```diff
@@ -0,0 +1,7 @@
+Gitlab.configure do |config|
```

Comment:
should be in the gitlab plugin ? (... and github should be in a github plugin)

Reply:
My patch from my working directory moved the GitLab config from where it was.  I will fix that.  There are still a couple of places in the Review process that reference GitHub, so the gem reference is still in the main application.